XamarinCommunityToolkit/Behaviors/ConfirmPasswordBehavior.cs

```diff
@@ -0,0 +1,22 @@
+using Xamarin.Forms;
+
+namespace Microsoft.Toolkit.Xamarin.Forms.Behaviors
+{
+    public class ConfirmPasswordBehavior : ValidationBehavior
+    {
+        public static readonly BindableProperty ComparedTextProperty =
+            BindableProperty.Create(nameof(ComparedText), typeof(string),
+                typeof(ConfirmPasswordBehavior), string.Empty);
+        protected override bool Validate(object value)
+        {
+            var confirmPasswordText = value as string;
```

Comment:
I'd suggest to use ToString method instead. It will allow using this Behavior with strings, numbers, guids etc.